#velocity3d

img_point[] pt
point3d[] vel

